SoftLayer/CLI/cdn/detail.py

```diff
@@ -9,24 +9,38 @@
 
 
 @click.command()
-@click.argument('account_id')
+@click.argument('unique_id')
+@click.option('--last_days',
+              default=30,
+              help='cdn overview last days less than 90 days, because it is the maximum e.g 7, 15, 30, 60, 89')
 @environment.pass_env
-def cli(env, account_id):
+def cli(env, unique_id, last_days):
     """Detail a CDN Account."""
 
     manager = SoftLayer.CDNManager(env.client)
-    account = manager.get_account(account_id)
+
+    cdn_mapping = manager.get_cdn(unique_id)
+    cdn_metrics = manager.get_usage_metrics(unique_id, days=last_days)
+
+    # usage metrics
+    total_bandwidth = str(cdn_metrics['totals'][0]) + " GB"
```

Comment:
I dont care for this style of string substituation. Lets go with 

`total_bandwidth = "%s GB" % cdn_metrics['totals'][0]` instead

do the same for total_hits and hit_ratio too

Reply:
Done